cvat-ui/src/components/annotation-page/standard3D-workspace/controls-side-bar/controls-side-bar.tsx

```diff
@@ -142,16 +133,51 @@ export default function ControlsSideBarComponent(props: Props): JSX.Element {
             <ObservedDrawCuboidControl
                 canvasInstance={canvasInstance}
                 isDrawing={activeControl === ActiveControl.DRAW_CUBOID}
-                disabled={!applicableLabels.length}
+                disabled={controlsDisabled}
             />
-            <ObservedGroupControl
-                switchGroupShortcut={normalizedKeyMap.SWITCH_GROUP_MODE}
-                resetGroupShortcut={normalizedKeyMap.RESET_GROUP}
+
+            <hr />
+
+            <ObservedMergeControl
+                mergeObjects={mergeObjects}
                 canvasInstance={canvasInstance}
                 activeControl={activeControl}
+                disabled={controlsDisabled}
+                shortcuts={{
+                    SWITCH_MERGE_MODE: {
+                        details: keyMap.SWITCH_MERGE_MODE,
+                        displayValue: normalizedKeyMap.SWITCH_MERGE_MODE,
+                    },
+                }}
+            />
+            <ObservedGroupControl
                 groupObjects={groupObjects}
-                disabled={!applicableLabels.length}
-                jobInstance={jobInstance}
+                resetGroup={resetGroup}
+                canvasInstance={canvasInstance}
+                activeControl={activeControl}
+                disabled={controlsDisabled}
+                shortcuts={{
+                    SWITCH_GROUP_MODE: {
+                        details: keyMap.SWITCH_GROUP_MODE,
+                        displayValue: normalizedKeyMap.SWITCH_GROUP_MODE0,
```

Comment:
Typo error here i suppose, leads to undefined in tooltip
![image](https://user-images.githubusercontent.com/50956430/215705897-59d4a3ea-db42-42ca-a995-beef42cba3b5.png)